### PR TITLE
lazy fetch of package details on UI Installed tab

### PR DESCRIPTION
**Pull request by johnataylor**
@alpaix as we discussed

basically immediately return the list of packages installed - with some details
and then lazily fetch the details from the sources

### CODE REVIEW

**Comment by johnataylor** [comment]
NuGet/Home#2089

**Comment by emgarten** [comment]
:shipit:

File: src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageMetadataProvider.cs [hunk 1]
```diff
@@ -115,6 +122,27 @@ internal class MultiSourcePackageMetadataProvider : IPackageMetadataProvider
                 .ToArray();
         }
 
+        private async Task<IEnumerable<VersionInfo>> FetchAndMergeVersionsAsync(PackageIdentity identity, bool includePrerelease, CancellationToken cancellationToken)
```
Comment (alpaix) [inline src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageMetadataProvider.cs:125]: :+1: for FetchAndMerge...

**Comment by alpaix** [comment]
looks great! :shipit:

**Pull request merged**